pkg/endpoint/endpoint.go

```diff
@@ -1768,7 +1768,9 @@ func (e *Endpoint) identityLabelsChanged(owner Owner, myChangeRev int) error {
 		return nil
 	}
 
-	if e.SecurityIdentity != nil && string(e.SecurityIdentity.Labels.SortedList()) != string(newLabels.SortedList()) {
+	if e.SecurityIdentity != nil &&
```

Comment:
Am I missing the change here or is this just a cosmetic change?

Reply:
`!=` -> `==`